Health check fails

### Behaviour

Cloudflared container runs normally but reports as unhealthy.

#### Steps to reproduce this issue

1. Pull my docker compose with your container https://github.com/grandegiraffo/pihole-doh
2. docker-compose up -d
3. Open Portainer or other tool to see health checks for all containers

#### Expected behaviour

> Tell me what should happen

#### Actual behaviour

> Tell me what happens instead

### Configuration

* Docker version (type `docker --version`) : version 20.10.6, build 370c289
* Docker compose version if applicable (type `docker-compose --version`) : version 1.21.0
* Platform (Debian 9, Ubuntu 18.04, ...) : Raspbian 
* System info (type `uname -a`) : Linux 5.10.17-v8+ #1421 SMP PREEMPT Thu May 27 14:01:37 BST 2021 aarch64 GNU/Linux
* Include all necessary configuration files : `docker-compose.yml`, `.env`, ...

### Docker info

```
> Output of command `docker info`
```

### Logs

```
> Container logs (set LOG_LEVEL to debug if applicable)
```

Status | unhealthy
Failure count | 21
Last output | time.c:126: Operation not permitted timer.c:634: fatal error: RUNTIME_CHECK(isc_time_now((&amp;now)) == 0) failed Aborted

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,7 @@
 ARG CLOUDFLARED_VERSION=2021.6.0
 
 FROM --platform=${BUILDPLATFORM:-linux/amd64} tonistiigi/xx:golang AS xgo
-FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.16-alpine3.13 AS builder
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.16-alpine3.14 AS builder
 
 COPY --from=xgo / /
 
@@ -21,7 +21,7 @@ ENV GO111MODULE=on
 ENV CGO_ENABLED=0
 RUN go build -v -mod vendor -ldflags "-w -s -X 'main.Version=${CLOUDFLARED_VERSION}' -X 'main.BuildTime=${BUILD_DATE}'" github.com/cloudflare/cloudflared/cmd/cloudflared
 
-FROM alpine:3.13
+FROM alpine:3.14
 LABEL maintainer="CrazyMax"
 
 ENV TZ="UTC" \
@@ -33,7 +33,7 @@ ENV TZ="UTC" \
 RUN apk --update --no-cache add \
     bind-tools \
     ca-certificates \
-    libressl \
+    openssl \
     shadow \
     tzdata \
   && addgroup -g 1000 cloudflared \

